examples/whattimeisitrightnow/tests/test_action_provider.py

```diff
@@ -31,14 +31,20 @@
 with mock.patch(
     "globus_action_provider_tools.authentication.TokenChecker.check_token",
     return_value=mock_authstate(),
-) as patched_check_token:
-    from examples.whattimeisitrightnow.app.app import app, schema
+):
+    from examples.whattimeisitrightnow.app.app import schema
 
 
 @pytest.fixture
 def client():
-    app.config["TESTING"] = True
-    yield app.test_client()
+    with mock.patch(
+        "globus_action_provider_tools.authentication.TokenChecker.check_token",
+        return_value=mock_authstate(),
+    ):
+        from examples.whattimeisitrightnow.app.app import app
```

Comment:
is this import here related to how the mock is getting installed? Its kinda a funky notation...

Reply:
Yea it has to do with how the `TokenChecker` gets instantiated at import time... The example could probably be refactored to have a slightly cleaner way of getting mocked but I was hoping not to spend too much time refactoring examples.